repos/fdbt-netex-output/src/netex-convertor/point-to-point-tickets/pointToPointTicketNetexHelpers.ts

```diff
@@ -100,7 +100,17 @@ export const getPointToPointScheduledStopPointsList = (fareZones: FareZone[]): S
         stops = getStops(copyOfFareZones);
     }
 
-    return stops.map(stop => ({
+    const set = new Set();
+
+    const stopsWithDuplicatesRemoved = stops.filter(x => {
```

Comment:
Can we call `x` as `stop` instead? Keeps it consistent with the naming on line 113